addons/l10n_ke/data/template/account.tax-ke.csv

```diff
@@ -1,97 +1,101 @@
-"id","description","invoice_label","type_tax_use","name","amount_type","amount","tax_group_id","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/factor_percent","repartition_line_ids/account_id"

```

Comment:
@dbkosky Is this diff entirely only about the item codes?

Reply:
yes, it should be